src/bindings/redux-form/Textarea.jsx

```diff
@@ -9,17 +9,28 @@ import FieldGroup from './FieldGroup';
  *
  * @see https://redux-form.com/7.0.4/docs/api/field.md/#2-a-stateless-function
  */
-export const TextareaField = ({ input, ...props }) => (
-  <FieldGroup {...props}>
-    <Forms.Textarea {...input}  />
-  </FieldGroup>
-);
+export const TextareaField = ({ input, ...props }) => {
+  const textareaProps = { autosize: props.autosize };
```

Comment:
Why not extracting autosize the line above to avoid passing it to FieldGroup?

Reply:
@jducro thats fair and I'm already doing it. :+1: